test/unit/errors.spec.js

```diff
@@ -1,36 +1,41 @@
 import postscribe from '../../src/postscribe';
 
-describe('errors', () => {
-  function handlesError(html) {
-    // Naming finish b/c the callback to postscribe is already called done
-    return finish => {
-      let oldOnError = window.onerror;
-      window.onerror = null;
-
-      const div = document.createElement('div');
-      document.body.appendChild(div);
-
-      postscribe(div, html, {
-        error(err) {
-          window.onerror = oldOnError;
-          expect(err).to.be.ok();
-          finish();
-        },
-        done() {
-          window.onerror = oldOnError;
-          finish();
-        }
-      });
-    };
-  }
-
-  it('syntax error', handlesError('<script>va x</script>'));
-
-  it('js exception', handlesError('<script>throw 1;</script>'));
-
-  // IE cannot report remote script errors
-  it('remote script malformed url', handlesError('<script src="404"></script>'));
-
-  it('remote script 404', handlesError('<script src="http://cdn.krxd.net/not_found"></script>'));
-
-});
+// TODO Skipping these on IE
+if (navigator.userAgent.indexOf('MSIE') === -1) {
```

Comment:
This is holdover from previous tests. Surprisingly, these mostly work, but the errors cause IE8 to stop what it's doing, which I think wasn't helping on sauce labs.